src/index.html

```diff
@@ -16,17 +16,40 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+      name="search-bar"
+      class="search-bar"
+    >
+      <label>
+        <input
+          type="text"
+          placeholder="Try “Los Angeles“"
+          data-qa="keypress"
+          name="search-bar-big"
+          class="search-bar__input search_bar__input--big"
+        />
+      </label>
+    </form>
+    <form
+      action="#"
+      method="post"
+      data-qa="small"
+      name="search-bar"
```

Comment:
The 'name' attribute for the second form is the same as the first form ('search-bar'). Each form should have a unique name attribute to avoid conflicts.